config.go

```diff
@@ -29,7 +29,7 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2020-04-27"
+	ClientVersion = "2020-04-28"
```

Comment:
Should update this again

Reply:
not yet done (I'll wait until it's ready to merge)